build.sbt

```diff
@@ -44,7 +44,9 @@ val junitDep          = "junit"                          % "junit"
 val junitInterfaceDep = "com.novocode"                   % "junit-interface"                  % "0.11"                            % "test"
 val jolDep            = "org.openjdk.jol"                % "jol-core"                         % "0.9"
 val asmDep            = "org.scala-lang.modules"         % "scala-asm"                        % versionProps("scala-asm.version")
-val jlineDep          = "jline"                          % "jline"                            % versionProps("jline.version")
+val jlineDep          = "org.jline"                      % "jline"                            % versionProps("jline.version")
+val jansiDep          = "org.fusesource.jansi"           % "jansi"                            % versionProps("jansi.version")
+val jnaDep            = "net.java.dev.jna"               % "jna"                              % versionProps("jna.version")
```

Comment:
https://github.com/jline/jline3#jansi-vs-jna says

>  If one of the Jansi or JNA library is present, it will be used and JLine will use native calls

> There is no difference between JLine's support for Jansi and JNA. Both will provide the exact same behaviors. So it's a matter of preference

So do we need both?

Reply:
I remember I tried the simple thing and it didn't work, but that was a long time ago. I'll try to refresh some memory cells and also experiment.

Reply:
It looks like coursier uses [just jansi](https://github.com/coursier/coursier/commit/e9d81d81130f0280adb39525e4b9dba7f9a3bd06) -- maybe this can inspire?

Reply:
the discussion continues at https://github.com/scala/scala-dev/issues/677